sdk/eventhub/Azure.Messaging.EventHubs/src/Metadata/PartitionConsumerMetrics.cs

```diff
@@ -0,0 +1,119 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Messaging.EventHubs.Metadata
+{
+    /// <summary>
+    ///   A set of information about the state of a partition, as observed by the consumer.
+    /// </summary>
+    ///
+    public class PartitionConsumerMetrics
+    {
+        /// <summary>
+        ///   The name of the Event Hub where the partitions reside, specific to the
+        ///   Event Hubs namespace that contains it.
+        /// </summary>
+        ///
+        public string EventHubName { get; }
+
+        /// <summary>
+        ///   The identifier of the partition, unique to the Event Hub which contains it.
+        /// </summary>
+        ///
+        public string Id { get; }
```

Comment:
I would suggest to rename this to `PartitionId`
Else, this looks like it is the "id" for "PartitionConsumerMetrics"

Reply:
Makes sense.  I like it.